File: .netWebApi/nemsport-web-api/Models/EventMember.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace nemsport_web_api.Models
{
    public class EventMember
    {
        public int Id { get; set; }
        public int Fk_member_id { get; set; }
        public int Fk_event_id { get; set; }

        EventMember()
        {

        }

        EventMember(int id, int fk_member_id, int fk_event_id)
        {
            Id = id;
            Fk_member_id = fk_member_id;
            Fk_event_id = fk_event_id;
        }

        public override string ToString()
        {
            return $"{nameof(Id)}: {Id}, {nameof(Fk_member_id)}: {Fk_member_id}, {nameof(Fk_event_id)}: {Fk_event_id}";

        }

    }
}

File: app/src/main/java/com/dtu/nemsport/models/EventMember.kt
package com.dtu.nemsport.models

data class EventMember(val id: Int,
                       val fk_member_id: Int,
                       val fk_event_id: Int) {

    override fun toString(): String {
        return  "id: $id - " +
                "fk_member_id: $fk_member_id - " +
                "fk_event_id: $fk_event_id"
    }
}
